src/client/src/tscLocal.c

```diff
@@ -382,10 +382,10 @@ static int32_t tscGetTableTagValue(SCreateBuilder *builder, char *result) {
 
     if (i == 0) {
       snprintf(result + strlen(result), TSDB_MAX_BINARY_LEN - strlen(result), "%s", "(");
-    } 
-    if ((fields[i].type == TSDB_DATA_TYPE_NCHAR 
-        || fields[i].type == TSDB_DATA_TYPE_BINARY 
-        || fields[i].type == TSDB_DATA_TYPE_TIMESTAMP) && 0 == ret) {
+    }
+    if ((fields[i].type == TSDB_DATA_TYPE_NCHAR || fields[i].type == TSDB_DATA_TYPE_BINARY ||
```

Comment:
It is simply an effect of git clang-format. No semantic or syntax changed at all.